packages/nc-gui-v2/plugins/init.ts

```diff
@@ -0,0 +1,46 @@
+import type { Dayjs } from 'dayjs'
```

Comment:
Don't name your plugin file `init`
It's not evident what `init` is supposed to do by looking at the name.
Please be specific what your plugin does and name it accordingly, in this case probably `feedbackForm` would probably suffice to hint to others what this plugin might do.

Reply:
@bcakmakoglu My idea was to have a plugin which calls these kind of little one off functions when the app initializes.

Reply:
Plugin name should still be updated. `Init` is just not a valid plugin name.